src/lib.rs

```diff
@@ -2,3 +2,19 @@
 pub mod matchspec;
 mod parsers;
 pub use crate::matchspec::*;
+
+use pyo3::prelude::*;
+use pyo3::wrap_pyfunction;
+
+
+#[pyfunction]
+fn match_against_matchspec(matchspec: String, package: String, version: String) -> bool {
+  let ms: matchspec::MatchSpec<String> = matchspec.parse();
+  ms.is_package_version_match(&package, &version)
+}
```

Comment:
Maybe the correct way to do this is by returning a `PyResult`? That way we don't lose the possible error. Maybe this?
```suggestion
fn match_against_matchspec(matchspec: String, package: String, version: String) -> PyResult<bool> {
  let ms: matchspec::MatchSpec<String> = matchspec.parse()?;
  Ok(ms.is_package_version_match(&package, &version))
}
```
That question mark after the `parse()` will return the error if there is one, otherwise will unwrap the `Ok(MatchSpec<String>)` and give just the `MatchSpec<String>`.

Reply:
I tried using `unwrap()` as suggested by @marcoesters (suggestion was made outside of this PR) and it seems to work!